src/ipz_parser.cpp

```diff
@@ -519,11 +517,11 @@ types::IPZVpdMap::mapped_type
                                               itrToKwds);
             kwdValueMap.emplace(std::move(kwdName), std::move(kwdValue));
         }
-
-        // support all the Keywords
-        auto stop = std::next(itrToKwds, kwdDataLength);
-        std::string kwdata(itrToKwds, stop);
-        kwdValueMap.emplace(std::move(kwdName), std::move(kwdata));
+        else
```

Comment:
why there is no support for other keywords which are not present in the supportedKeywords list?

why parse method parses and populates only specific keywords and not all the keywords on dbus?

Reply:
To handle all the keyword values for IPZ parser an issue is created - #221 . This will be taken care in that issue.

Reply:
ok